pkg/controller/core/workload_controller_test.go

```diff
@@ -285,13 +286,15 @@ func TestSyncCheckStates(t *testing.T) {
 
 	for name, tc := range cases {
 		t.Run(name, func(t *testing.T) {
-			gotStates, gotShouldChange := syncAdmissionCheckConditions(tc.states, tc.list)
+			gotStates, gotShouldChange := syncAdmissionCheckConditions(tc.states, sets.New(tc.list...))
 
 			if tc.wantChange != gotShouldChange {
 				t.Errorf("Unexpected should change, want=%v", tc.wantChange)
 			}
 
-			opts := []cmp.Option{}
+			opts := []cmp.Option{
+				cmpopts.SortSlices(func(a, b kueue.AdmissionCheckState) bool { return a.Name < b.Name }),
```

Comment:
Maybe we can always produce the list alphabetically?
Then it will be easier for users to find a particular check.

Reply:
Done